### DISCUSSION TITLE
New location for z_offset and a query about Save Z-Offset Macros

### DISCUSSION THREAD

**Discussion by steveharman** [Guilouz/Creality-Helper-Script-Wiki#480] [Issues]
This first part is just an observation....  Yesterday I updated my Max to the latest 1.3.3.8 firmware. Today I was looking through printer.cfg and noticed, I think, a change?

`[prtouch_v2] pr_version: 2 step_base: 2 z_offset: 0
`
Prior to the latest firmware I'm _fairly sure_  that z_offset was probed by the machine and lived in the auto-save block at the end of `printer.cfg` - does that sound right?

As z_offset now appears to be outside of the auto-save block of `printer.cfg` , does that mean the probe no longer establishes my offset and the printer is purely using a manually specified value of 0 ?

In any case, I updated to the latest script and installed Save Z-Offset Macros (I haven't used it before today) but when I used Fluidd to tune my z-offset, the value I ended up with in the Fluidd UI wasn't reflected in my `printer.cfg` - which kept `z_offset: 0` even after a restart of Klipper.  

Is that the expected result?  That the `z_offset:` in printer.cfg remains at 0 even after I update the z-offset via Fluidd and have Save Z-Offset Macros istaleld?  I have the correct value written to `variables.cfg` just not in `printer.cfg`

Thanks.

**Comment by Guilouz** [comment]
Z-offset macros doesn’t change z_offset value in printer.cfg, it save gcode_offset
